vaadin-accordion-flow-parent/vaadin-accordion-flow/src/main/java/com/vaadin/flow/component/accordion/AccordionPanel.java

```diff
@@ -119,4 +133,64 @@ public AccordionPanel(Component summary, Component... components) {
         this(summary);
         addContent(components);
     }
+
+    /**
+     * Sets the summary container component.
+     */
+    protected void setSummaryContainer() {
```

Comment:
```suggestion
    @Override
    protected void setSummaryContainer() {
```

Reply:
When adding `@Override` to the protected method, for some reason I'm getting this error:

```
[ERROR] /Users/serhii/vaadin/flow-components/vaadin-accordion-flow-parent/vaadin-accordion-flow/src/main/java/com/vaadin/flow/component/accordion/AccordionPanel.java:[139,5] method does not override or implement a method from a supertype
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project vaadin-accordion-flow: Compilation failure
```

Actually it compiles and works as expected without `@Override`, maybe it's not needed.